src/screens/DrawerScreens/LoginScreen/components/LoginForm.js

```diff
@@ -0,0 +1,131 @@
+import React, {useState} from 'react';
+import {Block} from '../../../../common/simpleComponents/Block';
+import {StyledText} from '../../../../common/simpleComponents/Text';
+import {Input} from '../../../../common/simpleComponents/Input';
+import {StyledButton} from '../../../../common/simpleComponents/Button';
+import ShowPassword from '../../../../assets/icons/showPassword.svg';
+import HidePassword from '../../../../assets/icons/hidePassword.svg';
+import {Formik} from 'formik';
+import {loginValidationSchema} from './loginValidationSchema';
+import {currentErrors} from './loginValidationSchema';
+
+export const LoginForm = () => {
+  const [issSecureEntry, setSecureEntry] = useState(true);
+  const onTogglePassword = () => {
+    setSecureEntry(prev => !prev);
+  };
+  const onSubmitForm = (data, {resetForm}) => {
+    console.log(data);
+    resetForm();
+  };
+  return (
+    <Block pt={'10%'} pl={'10%'} pr={'10%'}>
+      <Formik
+        initialValues={{
+          username: '',
+          password: '',
+        }}
+        validateOnMount={true}
+        onSubmit={onSubmitForm}
+        validationSchema={loginValidationSchema}>
+        {({
+          handleBlur,
+          handleChange,
+          handleSubmit,
+          values,
+          errors,
+          touched,
+          isValid,
+        }) => (
+          <>
+            <Block
```

Comment:
form item should be moved to a separate customisable  components (label, input and error component)   then this fields should be mapped  in a form. This should be done to avoid code duplication